### PR TITLE
A little polish to Polish translation

### PR DESCRIPTION
**Pull request by YourSenseiCreeper**

### CODE REVIEW

File: src/js/dialogs/sandbox.js [hunk 1]
```diff
@@ -727,56 +727,21 @@ exports.dialog = {
...
-        '[https://pcottle.github.io/learnGitBranching/?demo](https://pcottle.github.io/learnGitBranching/?demo)',
-        '',
-        'Masz dość tego okna? Dodaj `?NODEMO` do adresu URL, aby się go pozbyć, w ten sposób:',
-        '',
-        '[https://pcottle.github.io/learnGitBranching/?NODEMO](?NODEMO)'
-      ]
-    }
-  }, {
-    type: 'ModalAlert',
-    options: {
```
Comment (pcottle) [inline src/js/dialogs/sandbox.js:751]: so this modal alert left?

**Comment by pcottle** [comment]
Have one comment about that view that got removed but otherwise lg!

File: src/js/dialogs/sandbox.js (hunk 1 above)
Comment (YourSenseiCreeper) [reply src/js/dialogs/sandbox.js:751]: I saw en_US translation without that part, so I removed it thinking it's not actual anymore.
Would you like me to bring it back?

Comment (pcottle) [reply src/js/dialogs/sandbox.js:751]: Oh then that's totally fine -- sorry I didn't check if it was more consistent or not.

Thanks for all your contributions on the polish translation!

**Pull request merged**